### ISSUE TITLE
Laravel Blade PHP file

### ISSUE THREAD

**Issue by TomClarkson** [kemayo/sublime-text-2-goto-documentation#21]
Could you please add blade.php into the scope thanks.

**Comment by Saborknight** [comment]
Yes please!

**Comment by Saborknight** [comment]
I thought installing "Laravel Blade Highlighter" would do the trick. That package installs scopes `text.blade text.html.blade` for blade. I assumed it would then just a matter of adding the relevant doc URL to your settings for Blade. However, sadly Sublime Text doesn't catch the blade scope, but instead it listens to the HTML scope... I don't know why.

When I run `view.settings().get('syntax')` I get the correct response: `'Packages/Laravel Blade Highlighter/blade.sublime-syntax'`
However when I run `view.scope_name(view.sel()[0].begin()).rpartition('.')[2].strip()` I get the incorrect response `'html'`..............

Despite all of this... There isn't any blade documentation which separates into specific sections for the key functions... So what would have to be done is add the documentation without a query, which means the documentation website isn't able to know the word you are looking for...

So basically, due to the official docs not having concise documentation on a function by function basis... it may not be as effective as the rest of the languages this package supports.
